vm/interpreter/src/fvm/externs.rs

```diff
@@ -91,7 +94,7 @@ impl<DB: BlockStore> ForestExterns<DB> {
             resolve_to_key_addr(&state, &gbs, &worker).map_err(|e| anyhow::anyhow!("{}", e))?;
 
         let gas_used = tracker.borrow().gas_used();
-        Ok((addr, gas_used))
+        Ok((addr, gas_used.as_milligas()))
```

Comment:
This should be in gas units, not milligas. Rounding up is probably the right thing to do.

Reply:
So you want to divide (round up) by the right factor?

Reply:
Yes, use the `round_up` function.

Reply:
Done